lib/resty/cassandra/policies/lb/req_dc_rr.lua

```diff
@@ -47,7 +51,11 @@ function _M:init(peers)
 
   for i = 1, #peers do
     if type(peers[i].data_center) ~= 'string' then
-      error('peer '..peers[i].host..' data_center field must be a string')
+      log(WARN, _log_prefix, 'peer ', peers[i].host, ' data_center field must be a string')
+
+      if peers[i].err ~= '' then
+        log(WARN, _log_prefix, 'peer ', peers[i].host, ' ', peers[i].err)
```

Comment:
No need to log this here, the driver does it elsewhere already. None of the other LB policies log this either.